src/test/crimson/seastore/test_extmap_manager.cc

```diff
@@ -242,39 +257,83 @@ TEST_F(extentmap_manager_test_t, force_split_balanced)
       for (unsigned j = 0; j < 5; ++j) {
         [[maybe_unused]] auto addref = insert_extent(extmap_root, *t, lo, {lo, len});
         lo += len;
-        if ((i % 10 == 0) && (j == 3)) {
-          check_mappings(extmap_root, *t);
-        }
       }
       logger().debug("submitting transaction");
       tm->submit_transaction(std::move(t)).unsafe_get();
-      if (i % 50 == 0) {
+    }
+    replay();
+    auto t = tm->create_transaction();
+    int i = 0;
+    for (auto iter = test_ext_mappings.begin(); iter != test_ext_mappings.end();) {
+      auto [lo, ext] = *iter;
+      ++iter;
+      rm_extent(extmap_root, *t, lo, ext);
+      i++;
+
+      if (i % 10 == 0) {
+        logger().error("submitting transaction i= {}", i);
```

Comment:
please don't use `error()` for informative messages.

Reply:
updated.